spec/state_machines/workflows/ico_permitted_events_spec/ico_appeal_trigger_spec.rb

```diff
@@ -0,0 +1,228 @@
+require 'rails_helper'
+
+describe ConfigurableStateMachine::Machine do
+  context 'trigger ico workflow' do
+
+  ##################### MANAGER FLAGGED ############################
```

Comment:
Good stuff adding these missing tests in!!

Reply:
I think we're also missing the ico_event_spec if you ever want to add it in :)